### DISCUSSION TITLE
gsap extraPlugins congig error?

### DISCUSSION THREAD

**Discussion by rowild** [hypernym-studio/nuxt-gsap#50] [Q&A]
I just wanted to setup gsap with nuxt 3:

```sh
npx nuxi@latest init .
npm i -D @hypernym/nuxt-gsap
```

// nuxt.config.ts

```js
export default defineNuxtConfig({
  devtools: {   enabled: true   },
  modules: [
    "@hypernym/nuxt-gsap"
  ],
  gsap: {
    extraPlugins: {
      flip: true,
      scrollTrigger: true,
      ...
    },
```

The config complains:

```sh
Argument of type '{ devtools: { enabled: boolean; }; modules: string[]; gsap: { extraPlugins: { flip: boolean; scrollTrigger: boolean; observer: boolean; scrollTo: boolean; }; }; }' is not assignable to parameter of type 'InputConfig<NuxtConfig, ConfigLayerMeta>'.
  Object literal may only specify known properties, and 'gsap' does not exist in type 'InputConfig<NuxtConfig, ConfigLayerMeta>
```

![Screenshot 2023-07-13 at 09 39 04](https://github.com/hypernym-studio/nuxt-gsap/assets/213803/a84a5a81-85cd-46ab-a58d-ffa797511c0f)

What am I doing wrong?

**Comment by rowild** [comment]
Oh... I ran `npm i` again, that created the ts types, now gsap is accepted.
However, "registerEffects" now is unhappy...

![Screenshot 2023-07-13 at 09 59 17](https://github.com/hypernym-studio/nuxt-gsap/assets/213803/52a0c7db-7f41-462a-b1c2-2e8b58b436f8)

**Reply by ivodolenc** [reply]
Hi, I can't reproduce this with the [official](https://github.com/hypernym-studio/nuxt-gsap#register-effects) example, which works fine on my end.

Can you provide a code example (not an image but copyable code)?
